src/extension.ts

```diff
@@ -673,7 +673,18 @@ class ExtensionManager implements vscode.Disposable {
         }
         const drv = await cmt.getCMakeDriverInstance();
         const opts = drv ? drv.expansionOptions : undefined;
-        const env = await effectiveKitEnvironment(kit, opts);
+        let env: Map<string, string> = new Map<string, string>();
+        if (kit) {
+          env = await effectiveKitEnvironment(kit, opts);
+        } else if (cmt.configurePreset) {
+          for (const key in cmt.configurePreset.environment) {
+            const value = cmt.configurePreset.environment[key];
```

Comment:
Just double checking... The vcvars are set in here, right?  I think I saw code that populates the environment with the stuff we need to find cl.exe, but just wanted to make sure you saw that too.

Reply:
Yes, they are inserted to configurePreset.environment during expansion.

Reply:
So, nothing for me to change here, right?